conf.py

```diff
@@ -53,7 +53,7 @@
 # Add any paths that contain custom themes here, relative to this directory.
 
 # Links
-extlinks = {'codedir': ('https://github.com/ros-planning/moveit_tutorials/tree/master/doc/pr2_tutorials//%s', ''),
+extlinks = {'codedir': ('https://github.com/ros-planning/moveit_tutorials/tree/indigo-devel/doc/pr2_tutorials//%s', ''),
```

Comment:
Same as above. Also, you can just reference the values from the html_context dict like I did here
https://github.com/PickNikRobotics/moveit_tutorials/blob/7dcb56d657e6c7150fe7a557c28762f463e4040b/conf.py#L56